CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing an SCM provider
```

Comment:
I think that `Contributing an SCM provider` should be put down and be the last section before CI

Reply:
done